stable/prometheus-operator/values.yaml

```diff
@@ -254,6 +254,11 @@ alertmanager:
     ##
     configMaps: []
 
+    ## ConfigSecret is the name of a Kubernetes Secret in the same namespace as the Alertmanager object, which contains configuration for
+    ## this Alertmanager instance. Defaults to 'alertmanager-' The secret is mounted into /etc/alertmanager/config.
+    ##
+    configSecret:
```

Comment:
I believe the convention is to comment out this property if it is not set

Reply:
Fixed. I commented option.